src/_help/changes-management/integrations.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Integrations
+---
+
+- TOC
+{:toc}
+
+## Slack integration
+
+Integrating Bump.sh with Slack can be done in three steps.
```

Comment:
Once again, before explaining "how to", we must define the what/why. The how is the easy part, but why should we use Slack integration? -> to notify people working on an API directly, like another team (front), or to inform the company member of an update, etc...